submission/submitter/src/test/kotlin/ac/uk/ebi/biostd/submission/service/AccNoServiceTest.kt

```diff
@@ -20,26 +20,31 @@ import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.CsvSource
 
 private const val SUB_ACC_NO = "AAB12"
+private val ACC_NUM = AccNumber("AAB", "12")
 private const val SUBMITTER = "submiter@email.com"
 private const val PROJECT = "CC123"
 private const val PROJECT_PATTERN = "!{ABC-}"
 
 @ExtendWith(MockKExtension::class)
 class AccNoServiceTest(
-    @MockK private val accNoPatternUtil: AccNoPatternUtil,
     @MockK private val context: PersistenceContext,
     @MockK private val submissionQueryService: SubmissionQueryService,
     @MockK private val privilegesService: IUserPrivilegesService
 ) {
+    private val accNoPatternUtil: AccNoPatternUtil = AccNoPatternUtil()
     private val testInstance = AccNoService(context, submissionQueryService, accNoPatternUtil, privilegesService)
 
     @ParameterizedTest(name = "prefix is {0} and numeric value is {1}")
     @CsvSource(
-        "AA, 88, AA/AA0-99/AA88",
-        "AA, 200, AA/AAxxx200/AA200"
-    )
-    fun getRelPath(prefix: String, value: Long, expected: String) {
-        assertThat(testInstance.getRelPath(AccNumber(prefix, value))).isEqualTo(expected)
+        "S-DIXA-AN-002, S-DIXA-AN/002/S-DIXA-AN-002",
+        "S-BSST11, S-BSST/011/S-BSST11",
+        "S-DIXA-011, S-DIXA/011/S-DIXA-011",
+        "1-AAA, 1-AAA/000/1-AAA",
+        "S-SCDT-EMBOJ-2019-103549, S-SCDT-EMBOJ-2019/549/S-SCDT-EMBOJ-2019-103549",
+        "S-DIXA-AN-002, S-DIXA-AN/002/S-DIXA-AN-002",
+        "S-SCDT-EMBOR-2017-44445V1, S-SCDT-EMBOR-2017-44445V/001/S-SCDT-EMBOR-2017-44445V1")
```

Comment:
I think it would be good to add the test case `S-BSST2` that returns `S-BSST/002/S-BSST2`

Reply:
this one looks pretty much as `S-BSST11` one no?